packages/expo-updates/scripts/createManifest.js

```diff
@@ -12,6 +12,9 @@ const filterPlatformAssetScales = require('./filterPlatformAssetScales');
   const destinationDir = process.argv[4];
   const entryFile = process.env.ENTRY_FILE || 'index.js';
 
+  // remove any relative '..' segments from project root
+  projectRoot = resolve(projectRoot);
```

Comment:
@brentvatne after running these changes in a mono repo, I found that this was needed to get things working

Reply:
reading the full script, it looks like `projectRoot` isn't defined at this point

Reply:
whoops! must have copied it wrong